src/get-started/wordcount-example.md

```diff
@@ -578,15 +579,15 @@ public class DebuggingWordCount {
 
 #### Direct Runner
 
-If you execute your pipeline using `DirectRunner`, it will print the log messages directly to your local console.
+If you execute your pipeline using `DirectRunner`, it prints the log messages directly to your local console.
 
-#### Dataflow Runner
+#### Cloud Dataflow Runner
 
-If you execute your pipeline using `DataflowRunner`, you can use Stackdriver Logging. Stackdriver Logging aggregates the logs from all of your Dataflow job's workers to a single location in the Google Cloud Platform Console. You can use Stackdriver Logging to search and access the logs from all of the workers that Dataflow has spun up to complete your Dataflow job. Logging statements in your pipeline's `DoFn` instances will appear in Stackdriver Logging as your pipeline runs.
+If you execute your pipeline using `DataflowRunner`, you can use Stackdriver Logging. Stackdriver Logging aggregates the logs from all of your Cloud Dataflow job's workers to a single location in the Google Cloud Platform Console. You can use Stackdriver Logging to search and access the logs from all of the workers that Cloud Dataflow has spun up to complete your job. Logging statements in your pipeline's `DoFn` instances will appear in Stackdriver Logging as your pipeline runs.
 
-If you execute your pipeline using `DataflowRunner`, you can control the worker log levels. Dataflow workers that execute user code are configured to log to Stackdriver Logging by default at "INFO" log level and higher. You can override log levels for specific logging namespaces by specifying: `--workerLogLevelOverrides={"Name1":"Level1","Name2":"Level2",...}`. For example, by specifying `--workerLogLevelOverrides={"org.apache.beam.examples":"DEBUG"}` when executing this pipeline using the Dataflow service, Stackdriver Logging would contain only "DEBUG" or higher level logs for the package in addition to the default "INFO" or higher level logs.
+If you execute your pipeline using `DataflowRunner`, you can control the worker log levels. Cloud Dataflow workers that execute user code are configured to log to Stackdriver Logging by default at "INFO" log level and higher. You can override log levels for specific logging namespaces by specifying: `--workerLogLevelOverrides={"Name1":"Level1","Name2":"Level2",...}`. For example, by specifying `--workerLogLevelOverrides={"org.apache.beam.examples":"DEBUG"}` when executing a pipeline using the Cloud Dataflow service, Stackdriver Logging will contain only "DEBUG" or higher level logs for the package in addition to the default "INFO" or higher level logs.
```

Comment:
The "If you execute your pipeline using `DataflowRunner`, ..." seems overly specified (esp. given the section headings)

maybe "When executing using the `DataflowRunner`, ..."?

Reply:
Changed to "When executing your pipeline with the `DataflowRunner`" and removed repetitive use of this for the second paragraph